lib/basicauth.php

```diff
@@ -76,8 +76,7 @@ public function checkPassword($uid, $password) {
 		}
 		switch (substr($headers[$responseIdx], 9, 1)) {
 			case "2":
-				$this->storeUser($uid);
-				return $uid;
+				return $this->storeUser($uid);
```

Comment:
I guess we would need to add this change to every backend...

Reply:
It should have no direct impact on any other backends, because it returned the uid before and returns the uid after the change too... just slightly diffrent.

It would likely make sense to adapt this change for other backends too, but i currently dont have the resources to do this and test it for all other backend. (I just fixed the problem for our instance. :)

Reply:
okay I see; thank you very much!